tools/mtouch/mtouch.cs

```diff
@@ -470,6 +471,8 @@ public static string GetAotArguments (Application app, string filename, Abi abi,
 			args.Append (app.AotArguments);
 			if (llvm_only)
 				args.Append ("llvmonly,");
+			else if (interp)
+				args.Append ("interp,");
```

Comment:
You have both `MONO_AOT_MODE_INTERP` and `MONO_AOT_MODE_INTERP_LLVMONLY`, so is llvmonly and interp really mutually exclusive?

Reply:
The Interp doesn't work without some code gen and if we're doing llvmonly code gen it works in a different way.